noop/noop_sink.go

```diff
@@ -0,0 +1,41 @@
+package noop
+
+import (
+	"context"
+
+	"github.com/uw-labs/substrate"
+)
+
+// AsyncMessageSink does no operations on publish messages calls
+type AsyncMessageSink struct{}
+
+// NewAsyncMessageSink returns a pointer to a new AsyncMessageSink
+func NewAsyncMessageSink() (substrate.AsyncMessageSink, error) {
```

Comment:
To be consistent with the other constructors, the return signature has `error` although it is not required in this case.

Reply:
I don't think the error is necessary. You will get an error anyway when initialising the sink using url and it's a bit annoying checking an error that's always nil in case you are using it directly...